src/style/content/conversation/message-list.less

```diff
@@ -274,7 +274,6 @@
 // MESSAGE - BODY
 .message-body {
   position: relative;
-  padding-right: 40px;
```

Comment:
Same here - we already have one big whitespace at the right, we don't need this one anymore